frontend/src/libs/components/input/input.tsx

```diff
@@ -16,8 +16,11 @@ type Properties<T extends FieldValues> = {
   label?: string;
   name: FieldPath<T>;
   placeholder?: string;
-  type?: 'text' | 'email' | 'password';
+  type?: 'text' | 'email' | 'password' | 'number' | 'dropdown';
   isDisabled?: boolean;
+  min?: number;
+  max?: number;
+  step?: string;
```

Comment:
I guess it works same as min/max props
```suggestion
  step?: number;
```